db/migrate/20200603113832_add_last_name_to_books.rb

```diff
@@ -0,0 +1,9 @@
+class AddLastNameToBooks < ActiveRecord::Migration[6.0]
+  def change
+    add_column :books, :last_name, :string, null: false, default: ''
```

Comment:
I think it might make sense at this point to introduce a new entity `Author`. It is a bit strange to have `first_name` and `last_name` on a book. And I'd say this is not just a matter of renaming these two columns. Since these two fields are clearly describing the author, it would make sense for them to live in a separate table `authors`. This will also simplify things if you end up adding even more info about the author (or more author-related logic).

Reply:
So ideally, there would be a new model `Authors`, as well as the `authors` table of course. I don't think you need any views/controllers for this entity, but you will need to add the association between a book and an author.

Reply:
I think the scenario is pretty similar to the one described here: https://guides.rubyonrails.org/kindle/association_basics.html

However, this addition does require some refactoring. It's not much, but still maybe you want to do this separate from this PR and then update this PR to work with `authors`.